src/miner.cpp

```diff
@@ -526,6 +527,10 @@ void static BitcoinMiner(const CChainParams& chainparams)
                     // Changing pblock->nTime can change work required on testnet:
                     hashTarget.SetCompact(pblock->nBits);
                 }
+
+                /* If we updated the block above, we also have to update
+                   the auxpow object to match the new block hash.  */
+                CAuxPow::initAuxPow(*pblock);
```

Comment:
Do we need to call `initAuxPow()` inside this particular loop? I don't see any updates to `pblock` inside this particular loop that does not break at this level? Am I missing something?